backend/core/src/entity/User.ts

```diff
@@ -0,0 +1,17 @@
+import { Entity, PrimaryGeneratedColumn, Column, CreateDateColumn, UpdateDateColumn } from "typeorm"
+
+@Entity()
+class User {
+  @PrimaryGeneratedColumn("uuid")
```

Comment:
I'm going to file a separate issue about this for discussion, but I was sort of hoping that we could use nanoids as the native primary keys across the board. I don't think it's a merge blocker while we sort it out, whichh hopefully will be quickly.

Reply:
Ya, I'm pretty neutral on this, but definitely be good to be consistent.

Reply:
Captured in #348.